README.md

```diff
@@ -100,6 +100,10 @@ Parses metadata XML and extracts the following properties:
 * `encryptionCert`
 * `claimSchema` - an object hash of claim identifiers that may be provided in the SAML assertion
 
+#### Options parameter details:
```

Comment:
Could you remove this heading and make the options all one list?

Reply:
The bulleted list above is the structure of the return, this heading separates the return structure from the structure of the options argument.  Don't think we should combine the bulleted lists here, but happy to re-organize (maybe position the options above the return?)

Reply:
Oh okay, I follow. Perhaps we should move the list of options ahead of the list of return values? What do you think? This just struck me as being a little confusing.

Reply:
Yeah, it felt awkward to me both above and below.  Will relocate it though;  Might be more helpful to use jsdoc block comments above the functions to document the options better.  Could then take that generated docset and publish to GitHub pages for better clarity.